configs/common/modules.yaml

```diff
@@ -123,6 +123,10 @@ modules:
             'YAML_DIR': '{prefix}'
             'YAML_LIB': '{prefix}/lib'
             'YAML_INC': '{prefix}/include'
+      madis:
+        environment:
+          set:
+            'MADIS_ROOT': '{prefix}'
```

Comment:
Please see my comment in https://github.com/JCSDA/spack/pull/311#issuecomment-1694030175

Reply:
I'm looking into how much trouble it will be if we needed to chance this. Many of the builds in operations don't use cmake, and the all-caps names are the convention. If it ends up being difficult to change in the near term, I'll probably ask that we take this approach for now since it's low risk, then make it a long-term goal to remove it.

Reply:
Yeah, I guess it's not feasible in the short term. As long as we add them manually and not change the standard in spack-stack that's fine with me, too.